### ISSUE TITLE
Analyze `clean_label()` results (Roman numerals, etc)

### ISSUE THREAD

**Issue by joeflack4** [monarch-initiative/omim#163]
## Overview
In #142, Joe created some outputs that show the results of labels before and after `clean_label()`.

Does it actually do what we want?
When it changes things to roman numerals, does that make things confusing in some cases? Like sometimes does it actually not look like a roman numeral number, but an acronym? E.g. "1A" changes to "IA"

## Resources
- [Google sheet](https://docs.google.com/spreadsheets/d/1cYuFYKEJ7nQoKzluz9uvOyEgeRr8GSj0kNooMBMXf8Q/edit?gid=1699952229#gid=1699952229)
- [Slack](https://monarchinitiative.slack.com/archives/C03AW548J8M/p1726369385994859)

**Comment by joeflack4** [comment]
@twhetzel I added to your board, but FYI your board doesn't have an "urgency" field for "low".

**Comment by matentzn** [comment]
This may not be that urgent, but I would like to suggest that

1. Numeric normalisation like this should be generalised and not be done ad-hoc just for OMIM ingest (no more scripts - oak synonymiser ideally or if this is too much of a lift mondolib)
2. This needs to be really carefully reviewed by a domain expert curator

**Comment by joeflack4** [comment]
@matentzn I agree with that. I've never used synonymiser but that sounds like a great idea! Just wanted to drop FYI just in case you were unaware: that this isn't a review of something new, but part of the pipeline that's been around for about 3 years.
